db/src/migrations/20240304143626_tags_releases.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+export const up = function (knex) {
+  return Promise.all([
+      knex.schema.createTable('tags_releases', table => {
+          table.primary(['tagId', 'releaseId']);
+          table.integer('releaseId')
+              .unsigned()
+              .references('id')
+              .inTable('releases')
+              .onDelete('CASCADE')
+              .index();
+          table.integer('tagId')
+              .unsigned()
+              .references('id')
+              .inTable('tags')
+              .onDelete('CASCADE')
+              .index();
```

Comment:
@facing-n we should make it good practice to put in created_at/updated_at columns in the tables that we create. it makes triaging any data issues or bugs far easier. here's an example: https://github.com/nina-protocol/nina-email-service/blob/main/db/src/migrations/20240123222342_create_latest_txid_table.js#L4

Reply:
good catch